claragenomics/dl4atac/layers.py

```diff
@@ -39,15 +39,15 @@ def __init__(self, interval_size, kernel_size, stride, dilation):
     @staticmethod
     def _get_total_same_padding(interval_size, kernel_size, stride, dilation):
         """
-        Describe.
+        Calculate total required padding.
 
         Args:
-            interval_size : 
-            kernel_size : 
-            stride :
-            dilation : 
+            interval_size : Genome interval size.
+            kernel_size : Size of filter.
+            stride : Stride for filter.
+            dilation : Filter dilation.
         Return:
-            required_total_padding :
+            required_total_padding
```

Comment:
description about what required_total_padding means might be necessary ?

Reply:
made it a bit more verbose